main/src/ca/uwaterloo/flix/language/phase/HtmlDocumentor.scala

```diff
@@ -220,7 +225,12 @@ object HtmlDocumentor {
     val sortedDefs = mod.defs.sortBy(_.sym.name)
 
     sb.append(mkHead(moduleName(mod)))
-    sb.append("<body>")
+    sb.append("<body class='no-script'>")
+
+    sb.append("<button id='theme-toggle' disabled aria-describedby='no-script'>")
+    sb.append("<span>Toggle theme.</span>")
+    sb.append("<div role='tooltip' id='no-script'>Enable JavaScript to use this toggle</div>")
```

Comment:
I find this text a bit long. Perhaps we can just skip it? So what if there is a "sun" you cannot click on?

Reply:
I don't see how it hurts, but I guess we could shorten it down to "Requires JavaScript"?

Reply:
Yes, lets do that 👍